crates/uv/src/commands/tool/run.rs

```diff
@@ -214,7 +216,28 @@ pub(crate) async fn run(
     // signal handlers after the command completes.
     let _handler = tokio::spawn(async { while tokio::signal::ctrl_c().await.is_ok() {} });
 
-    let status = handle.wait().await.context("Child process disappeared")?;
+    let mut term_signal = signal(SignalKind::terminate())?;
+    let mut int_signal = signal(SignalKind::interrupt())?;
```

Comment:
I think you need to remove the above `ctrl_c()` handler if we're going to replicate it here, right?